plugins/c8y_remote_access_plugin/src/input.rs

```diff
@@ -79,7 +82,13 @@ impl TryFrom<C8yRemoteAccessPluginOpt> for Command {
             C8yRemoteAccessPluginOpt {
                 connect_string: Some(message),
                 ..
-            } => Ok(Command::SpawnChild(message)),
+            } => {
+                if Path::new(UNIX_SOCKFILE).exists() {
```

Comment:
The test `input::tests::parses_spawn_child_if_no_flag_is_provided` just stopped working for me because of this branch:

```
failures:

---- input::tests::parses_spawn_child_if_no_flag_is_provided stdout ----
thread 'input::tests::parses_spawn_child_if_no_flag_is_provided' panicked at plugins/c8y_remote_access_plugin/src/input.rs:164:9:
assertion `left == right` failed
  left: TryConnectUnixSocket("530,jrh-rc-test0,127.0.0.1,22,cd8fc847-f4f2-4712-8dd7-31496aef0a7d")
 right: SpawnChild("530,jrh-rc-test0,127.0.0.1,22,cd8fc847-f4f2-4712-8dd7-31496aef0a7d")
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    input::tests::parses_spawn_child_if_no_flag_is_provided
```

I have `/run/c8y-remote-access-plugin.sock` file in my filesystem, and I don't know if it should be there or if some other test perhaps created it.

Also I think it's a bit unexpected for `TryFrom` impl to be doing side effects.

Reply:
Exactly because you have `/run/c8y-remote-access-plugin.sock` file in the device that is running test, it failed. The test should be modified to pass both case.

Reply:
Fixed by #3041